server/src/main/java/au/org/aodn/ogcapi/server/core/util/BboxUtils.java

```diff
@@ -0,0 +1,64 @@
+package au.org.aodn.ogcapi.server.core.util;
+
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.geom.LinearRing;
+import org.locationtech.jts.geom.Polygon;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Utility for BBox operation only, it assumes the incoming is a bounding box
+ */
+public class BboxUtils {
+    /**
+     * Normalize a bbox by adjusting longitudes to the range [-180, 180], and then split it into two if it
+     * is cross meridian. We need this because Bbox is assumed to work within this range only.
+     * @param minx - left
+     * @param maxx - right
+     * @param miny - top
+     * @param maxy - bottom
+     * @return - Geometry which is bounded [-180, 180]
+     */
+    public static Geometry normalizeBbox(double minx, double maxx, double miny, double maxy) {
+        // Bounding check, if greater than 360 already cover whole world, so adjust the maxx to something
+        // meaningful, noted that minx and maxx is not normalized yet so can be anything even beyond 180
+        if((maxx - minx) >= 360) {
+            // Value does not matter, it covered whole world which is equal to
+            minx = -180;
+            maxx = 180;
+        }
+        minx = (minx < -180) ? 180 - Math.abs(180 + minx) : minx;
+        maxx = (maxx > 180) ? -(maxx - 180) : maxx;
```

Comment:
there is a math formula for this: https://www.movable-type.co.uk/scripts/latlong.html, so we can avoid `?` comparing logics

> 	The longitude can be normalised to −180…+180 using (lon+540)%360-180

Reply:
I tried to use this formula but the test will failed with the number off slightly